base-math/trunctdsf.c

```diff
@@ -38,15 +41,28 @@ CONVERT_WRAPPER(
 	SRC_TYPE a_norm;
 	long long mant;
 	int	exp, sexp;
-	
+
```

Comment:
Same comment about unrelated changes.

Reply:
Same thing about differences between trunc*** files. Is this tab incorrect?